draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -811,6 +805,29 @@ Content-Type: text/plain
 </t>
 </section>
 
+<section title="Fragment Identifiers on http(s) URIs" anchor="uri.fragment.identifiers">
+  <iref item="Fragment Identifiers"/>
+<t>
+   Fragment identifiers allow for indirect identification
+   of a secondary resource, independent of the URI scheme, as defined in
+   <xref target="RFC3986" x:fmt="of" x:sec="3.5"/>.
+   Only some URI-based HTTP protocol elements allow the inclusion of fragment
```

Comment:
Some protocol elements that refer to a URI allow inclusion of a fragment, while others do not.

Reply:
They are distinguished by use of the ABNF rule <URI-reference> for elements where fragment is allowed; otherwise, a specific rule that excludes fragments is used (e.g., target-resource).